api/v1/peer/peer.proto

```diff
@@ -0,0 +1,50 @@
+// Copyright 2020 Authors of Hubble
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+
+import "google/protobuf/empty.proto";
+
+package peer;
+
+// Peer lists  hubble peers and notifies of changes.
+service Peer {
+    // Notify sends information about hubble peers in the cluster.
+    // When Notify is called, it sends information about all the peers that are
+    // already part of the cluster (with the type as PEER_ADDED). It
+    // subsequently notifies of any change.
+    rpc Notify(google.protobuf.Empty) returns (stream ChangeNotification) {}
+}
+
+// ChangeNotification indicates a change regarding a hubble peer.
+message ChangeNotification {
+    // Name is the name of the peer, typically the hostname. The name includes
+    // the cluster name if a value other than default has been specified.
```

Comment:
nit: Maybe clarify that this uniquely identifies a host. An example would also be nice (i.e. how does it include the cluster name).